project/build.scala

```diff
@@ -80,19 +88,22 @@ object BananaRdfBuild extends Build {
   lazy val ntriplesM  = CrossModule(crossBuildType,
     id                = "ntriples",
     baseDir           = "io/ntriples",
-    defaultSettings   = buildSettings,
+    defaultSettings   = buildSettings++ Seq(aggregate in Test := false),
```

Comment:
right - what does `aggregate in Test := false` do ? it means "if test is called on this project, don't also call test in the dependOn/aggregate projects". So:
- if this was moved to `buildSettings`, nothing would be tested
- it it was removed everywhere, everything would of course work. But that's not the point.

The reason for having this is that we have projects that have no tests (eg rdf) there is no point trying to test them. Sure we _can_ - we just  get
```sbt
ScalaCheck
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
[info] ScalaTest
[info] Run completed in 1 second, 27 milliseconds.
[info] Total number of tests run: 0
[info] Suites: completed 0, aborted 0
[info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
```

But the reason I added this was not just for niceness - a jvm gets fired up for each test project, potentially in parallel, causing memory issues. 

A general note here is that in _most_ scala projects, the test ans main 

A general note here is that in _most_ scala projects, the test and main code live together. And normally don't have projects that are pure abstractions. These are the two main factors that make b-rdf, not unique, but slightly different